src/navigation/sectionalTabs/sectionalTabContent.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /**
+     * Children to be rendered when the getContent function is undefined/null.
+     */
+    children: PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string]),
+    /**
+     * Class names to potentially override styling.
+     */
+    classNames: PropTypes.string.isRequired,
+    /**
+     * Main function to get the content, is passed from the <SectionalTabs /> items objects.
+     * It could return a string, an object or an array.
+     */
+    getContent: PropTypes.func,
+    /**
+     * Tab content component identifier.
+     */
+    id: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+    children: undefined,
+    getContent: undefined,
+};
+
+function SectionalTabContent(props) {
```

Comment:
Do we need a component description doc block comment on this guy?

Reply:
This component is not being exposed in the DOCs through the **react-docgen-loader**, whould it be neccesary to do it @morethanfire ?

Reply:
Probably not if it's not going to be used to power documentation.  Don't mind me!  😁